docs/docs/testkube-cloud/articles/architecture.md

```diff
@@ -5,8 +5,9 @@
 The main Testkube Cloud feature is to have insights into multiple Testkube Cloud Agents. 
 You can look at your Kubernetes clusters from a single dashboard and easily switch between different Testkube clusters. 
 
+![multiple clusters](../../img/multiple-clusters.jpeg)
 
-![multiple clusters](https://user-images.githubusercontent.com/30776/208391158-a42d1f56-950f-48c3-bcfb-2768054b4704.jpeg)
+<!-- ![multiple clusters](https://user-images.githubusercontent.com/30776/208391158-a42d1f56-950f-48c3-bcfb-2768054b4704.jpeg) -->
```

Comment:
I see you've commented out this line in all the files. 
Could you please remove it?